geode-web/src/test/java/org/apache/geode/management/internal/cli/commands/ConnectCommandWithSSLTest.java

```diff
@@ -156,10 +160,25 @@
   @Rule
   public RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
 
+  @Rule
+  public GfshShellConnectionRule gfsh = new GfshShellConnectionRule();
+
   private static MemberVM locator;
   private OutputStream out = null;
   private File sslConfigFile = null;
 
+  /*
+   * Tests are susceptible to failure if the DUnit VMs are left in an indeterminate state by
+   * previously run tests. Bouncing the VMs here ensures that this class starts with a clean
+   * environment. The Rules used here will leave the DUnit VMs clean for subsequent tests.
+   */
+  @BeforeClass
```

Comment:
you don't need to do this. This class already has a CleanupDUnitVMsRule classRule that made sure the vms are clean AFTER the tests are run. You can add a before in that rule to call the after which will make sure the vms are clean BEFORE the tests are run too. And this will help all tests that are using the rule as well.

Reply:
I was just typing up the same comment :)

Reply:
Yes, makes more sense that way. Thanks.